ENVParser/Program.cs

```diff
@@ -1,54 +1,53 @@
 ﻿
 using ENVParser;
-using static ENVParser.DataDictionary;
+using ENVParser.Utils;
 internal class Program
 {
     private static void Main(string[] args)
     {
         if (args.Length == 0)
         {
             Console.WriteLine("\nUsage: ENVParser <file path> <csv|json>(optional)");
-            Console.WriteLine("\n\t - Please either drop an env file on the executable or provide an ENV file path as a command-line argument.");
-            Console.WriteLine("\t - Optionally, provide an output file type as the second argument (json or csv).");
+            Console.WriteLine("\n\t - Either drop an ENV or provide a file path as a command-line argument.");
+            //Console.WriteLine("\t - If proving an ENV file, you can optionally specify an output file type as the second argument.");
```

Comment:
Are you intending to uncomment these when you add the functionality back in?

Reply:
Yes, the JSON and CSV implementations restore these lines